src/util.h

```diff
@@ -558,11 +558,12 @@ enum ERecorderState
 enum EChSortType
```

Comment:
Not part of this PR, but having this enum in utils looks unorganized.

Reply:
Yeah.  I'll look to move it.

Reply:
OK it's in util.h because it's used in CSettings as well as the mixer board, I think -- all of the `static_cast<ENUM>` enums that CSettings does get shared with somewhere else and CSettings doesn't want to be fishing through lots of headers, so they get dumped into util.h.

(Ideally, CSettings wouldn't work like that: each component would have its own settings and read or write them to the XML that CSettings simply read from a file or wrote to a file...  But _that_ is definitely a different PR!)